sagemaker-pipelines/tabular/caching/sagemaker-pipelines-caching-steps.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>It would be better to add a screenshot here to display what a cache hit/miss looks like in the studio UI, for better user experience.</p><p> </p><p>If the release date is too close, we can mark this as a follow up item.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>